internal/gatewayapi/listener_test.go

```diff
@@ -45,7 +45,8 @@ func TestProcessTracing(t *testing.T) {
 		},
 	}
 
-	for _, c := range cases {
+	for i := range cases {
```

Comment:
prefer if kept the style same across entire code case and used the existing style of reassigning the var inside the loop i.e.
```
for _, c := range cases {
    c := c
```

Reply:
updated